src/components/common/SafeIcon/index.tsx

```diff
@@ -1,29 +1,40 @@
 import type { ReactElement } from 'react'
-import { Box } from '@mui/material'
+import { Box, Tooltip } from '@mui/material'
 
 import css from './styles.module.css'
 import Identicon, { type IdenticonProps } from '../Identicon'
+import classnames from 'classnames'
 
 interface ThresholdProps {
   threshold: number | string
   owners: number | string
 }
 const Threshold = ({ threshold, owners }: ThresholdProps): ReactElement => (
-  <Box className={css.threshold} sx={{ color: ({ palette }) => palette.static.main }}>
+  <Box className={classnames(css.badge, css.threshold)} sx={{ color: ({ palette }) => palette.static.main }}>
     {threshold}/{owners}
   </Box>
 )
 
+const PendingActionsBadge = ({ queuedTxs }: { queuedTxs: string }): ReactElement => (
+  <Tooltip title="queued transactions at this Safe" placement="top">
```

Comment:
`at Safe` is not grammatically correct.

```suggestion
  <Tooltip title={`${queuedTxs} pending transactions`} placement="top">
```